WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -4531,11 +4531,12 @@
 				7EFF207F20EAC59D009C4699 /* Groups */,
 				7E7947A7210BABF5005BB851 /* Ranges */,
 				7E3E7A5E20E44E110075D159 /* Styles */,
+				CE46018A21139E8300F242B6 /* FooterTextContent.swift */,
 				7EFF208B20EADF68009C4699 /* FormattableCommentContent.swift */,
 				7EFF208520EAD918009C4699 /* FormattableUserContent.swift */,
 				73BFDA89211D054800907245 /* Notifiable.swift */,
 				7EB5824620EC41B200002702 /* NotificationContentFactory.swift */,
-				CE46018A21139E8300F242B6 /* FooterTextContent.swift */,
```

Comment:
This value was moved to `4534` above, as the directory contents were alphabetized in the branch.